powerauth-webflow-client/pom.xml

```diff
@@ -36,6 +36,16 @@
     <dependencies>
 
         <!-- Spring Boot -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
I am not sure about this.

Reply:
I mean exclusion of `log4j-to-slf4j`.

Reply:
@banterCZ I have removed the exclusions of log4j-to-slf4j from other poms as well

Reply:
🤔 We should revisit all logging dependencies in another issue.